strawberryfields/ops.py

```diff
@@ -305,6 +305,7 @@ def square(q):
 
 # numerical tolerances
 _decomposition_merge_tol = 1e-13
```

Comment:
is this tolerance no longer used?

Reply:
It is, but just for merges. I added _decomposition_tol for all the other checks.